modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItRaftCommandLeftInLogUntilRestartTest.java

```diff
@@ -175,9 +174,9 @@ private void restartClusterWithNotAppliedCommands(
         BinaryRowEx key = new TupleMarshallerImpl(table.schemaView()).marshal(Tuple.create().set("id", 42));
 
         if (isNode0Leader) {
-            assertNull(table.internalTable().get(key, new HybridClockImpl().now(), node1.node()).get());
+            assertNull(table.internalTable().get(key, node1.clock().now(), node1.node()).get());
         } else {
-            assertNull(table.internalTable().get(key, new HybridClockImpl().now(), node0.node()).get());
+            assertNull(table.internalTable().get(key, node1.clock().now(), node0.node()).get());
```

Comment:
Should we use node0.clock().now() here or using node1 is fine?

Reply:
Here it has no difference.
The only thing that matters is that the data was applied finally.